Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -164,11 +196,76 @@ private void CreateZoneMusic(PxVobZoneMusicData vob)
             soundCreator.Create(vob, parentGos[vob.type]);
         }
 
+        /// <summary>
+        /// Basically a free point where NPCs can do something like sitting on a bench etc.
+        /// @see for more information: https://ataulien.github.io/Inside-Gothic/objects/spot/
+        /// </summary>
+        private void CreateSpot(PxVobData vob)
```

Comment:
Add Debug Option to enable/disable the spots from rendering

Reply:
It's there. Just look into the method. It even adds labels if we're in Editor mode.

Reply:
Added comment for better visibility.